README.md

```diff
@@ -22,14 +22,15 @@ Or install it yourself as:
 
 ## Usage
 
-Add each calculated attribute to your model using the `calculated` keyword. It accepts two parameters: a symbol representing the name of the calculated attribute, and a lambda containing a string to calculate the attribute.
+Add each calculated attribute to your model using the `calculated` keyword. It accepts two parameters: a symbol representing the name of the calculated attribute, and a lambda containing a string to calculate the attribute. The lambda can accept arguments.
 
 For example, if we have two models, `Post` and `Comment`, and `Comment` has a `post_id` attribute, we might write the following code to add a comments count to each `Post` record in a relation:
 
 ```ruby
 class Post < ActiveRecord::Base
 ...
   calculated :comments_count, -> { "select count(*) from comments where comments.post_id = posts.id" }
+  calculated :comments_count_by_user, ->(user) { "select count(*) from comments where comments.post_id = posts.id and posts.user_id = #{user.id}" }
```

Comment:
I think we ought to avoid using string interpolation here as it potentially opens the user to SQL injection, and ActiveRecord typically handles parameter escaping so the user would not necessarily suspect that they need to sanitize their variables. This can be done with the `sanitize_sql` method on `ActiveRecord::Base` -- the lambda can return an array (first value the SQL and subsequent values any parameters to insert) and then in [model_methods.rb#79](https://github.com/lowjoel/calculated_attributes/blob/8b889205770f69912798369d0215b6215f608ecc/lib/calculated_attributes/model_methods.rb#L79) you can do e.g. 

```ruby
sql_parts = lam.call(*arguments)
sql = klass.send(:sanitize_sql, *sql_parts)
```